dev/NavigationView/NavigationView.cpp

```diff
@@ -340,17 +339,6 @@ void NavigationView::OnApplyTemplate()
         backButtonToolTip.Content(box_value(navigationBackButtonToolTip));
     }
 
-    if (auto buttonHolderGrid = GetTemplateChildT<winrt::Grid>(c_buttonHolderGrid, controlProtected))
```

Comment:
>buttonHolderGrid [](start = 13, length = 16)

Looks like the template part ButtonHolderGrid isn't needed now?

Reply:
It's still used by layout to figure out the top nav's width.